token/cli/src/config.rs

```diff
@@ -280,9 +283,29 @@ impl<'a> Config<'a> {
                 (default_program_id, false)
             };
 
+        // need to specify a compute limit if compute price and blockhash are specified
+        if matches.is_present(BLOCKHASH_ARG.name)
+            && matches.is_present(COMPUTE_UNIT_PRICE_ARG.name)
+            && !matches.is_present(COMPUTE_UNIT_LIMIT_ARG.name)
+        {
+            eprintln!(
+                "error: need to set {} if {} and {} are set",
+                COMPUTE_UNIT_LIMIT_ARG.name, COMPUTE_UNIT_PRICE_ARG.name, BLOCKHASH_ARG.name,
+            );
+            exit(1);
```

Comment:
Nit: this works, but it might be nice to use a clap error to emulate clap `requires` handling with the error output. Similar to what we do here: https://github.com/anza-xyz/agave/blob/ccb0b55d82fa40041cfc4d346ff50486c7195b7a/validator/src/main.rs#L911-L916
Wdyt?

Reply:
Great idea, done!